website/src/pages/case-studies/sofi-moves-fast-on-hundreds-of-microservices.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import { CaseStudyPage, CaseStudyRequestDemoForm, InContentBlockquote } from '../../components/content/CaseStudyPage'
+import { ContentSection } from '../../components/content/ContentSection'
+import Layout from '../../components/Layout'
+
+export default ((props: any) => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'SoFi moves fast on hundreds of microservices with Sourcegraph',
+            description:
+                'With Sourcegraph, SoFi can innovate and move quickly while keeping up with hundreds of microservices.',
+            image: 'https://localhost:9000/case-studies/sofi-sourcegraph-case-study-og-embed.png',
+        }}
+    >
+        <CaseStudyPage
+            customer="SoFi"
+            title="SoFi moves fast on hundreds of microservices with Sourcegraph"
+            logo="/case-studies/sofi-logo.png"
+            quote={{
+                quote:
+                    'Sourcegraph is an ingrained part of my daily process at SoFi. With Sourcegraph, our engineers can understand all of the repercussions of committing a change to a service that is exposed to other services.',
+                author: 'Ursula Robertson, Engineering Manager, SoFi',
+                image: '/case-studies/ursula-robertson-sofi.jpg',
+            }}
+            // pdf="https://storage.googleapis.com/sourcegraph-assets/SoFi-Sourcegraph-case-study.pdf"
+        >
+            <ContentSection color="white" className="col-md-6">
+                <div className="container">
+                    <p>
+                        SoFi helps its over 900,000 members achieve financial independence to realize their ambitions
+                        with products for borrowing, saving, spending, investing, and protecting their money. With
+                        Sourcegraph, SoFi can innovate and move quickly while keeping up with hundreds of microservices.
+                    </p>
+                    <h2 className="pt-5 pb-1">The need for cross-repository code search</h2>
+                    <p>
+                        When SoFi (Social Finance Inc.) decided to switch their codehost from Bitbucket to GitLab
```

Comment:
From the [style guide](https://about.sourcegraph.com/handbook/communication/style_guide#specific-terms-in-prose): prefer "code host" over "codehost".

```suggestion
                        When SoFi (Social Finance Inc.) decided to switch their code host from Bitbucket to GitLab
```

Reply:
Good call!